src/main/scala/org/broadinstitute/dsde/firecloud/core/ProfileClient.scala

```diff
@@ -101,122 +85,27 @@ class ProfileClientActor(requestContext: RequestContext) extends Actor with Fire
       } recover { case e: Throwable => RequestCompleteWithErrorReport(InternalServerError,
         "Unexpected error updating NIH link", e) }
       // Complete syncWhitelist and ignore as neither success nor failure are useful to the client
-      syncWhiteListResult onComplete {
-        case _ => profileResponse pipeTo parent
-      }
+      syncWhiteListResult pipeTo sender
```

Comment:
The comment just above this line says, "Complete syncWhitelist and ignore as neither success nor failure are useful to the client." I also tested linking on dev. Maybe this shouldn't be piped anywhere?